### ISSUE TITLE
.[Strict Warning (162)] File(kolbot/libs\ntitemparser.dbl:383) reference to undefined property p_section[i][p_end] 

### ISSUE THREAD

**Issue by pivoivo** [kolton/d2bot-with-kolbot#753]
Hey guys,

I always get that error: .[Strict Warning (162)] File(kolbot/libs\ntitemparser.dbl:383) reference to undefined property p_section[i][p_end] 

never changed anything in that data. Rest is running well.
mybe someone could help
Kind regards

**Comment by noah-** [comment]
did you modify any of your nip files? seems like there is an error in one of them

**Comment by pivoivo** [comment]
No, never opend it. thats why i was wondering.  just changed some stuff in picket etc.

**Comment by cameronpfs** [comment]
.nip is your pickit files... It is where your item pickup related files are.
